api/v1/serializers/parks-serializer.js

```diff
@@ -0,0 +1,84 @@
+const appRoot = require('app-root-path');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { apiBaseUrl, resourcePathLink, paramsLink } = appRoot.require('utils/uri-builder');
+
+const parkResourceProp = openapi.definitions.ParkResource.properties;
+const parkResourceType = parkResourceProp.type.enum[0];
+const parkResourceKeys = _.keys(parkResourceProp.attributes.properties);
+parkResourceKeys.push('owner'); // for relationships to work correctly
+const parkResourcePath = 'parks';
+const parkResourceUrl = resourcePathLink(apiBaseUrl, parkResourcePath);
+
+const getAmenitiesArray = (rawPark) => {
+  const listOfAmenities = openapi.parameters.filterAmenitiesAll.items.enum;
```

Comment:
Could you pull this list from the amenities enum in `ParkResource` instead? This shouldn't change anything, but this will make sure that the enum is coming from the resource instead of the filter parameter